### ISSUE TITLE
[TTY]: Facility Locator - Search results should have TTY: 711 links for deaf/hard-of-hearing users

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-cms#13939]
## User Story or Problem Statement
As a user who might want to make a Text Telephone (TTY) call, I would like to have that link available on the Facility Locator search results. If I don't have TTY set up on my smart phone, I would have to exit the Facility Locator application to call 7-1-1.

## Details
I've been working with Meli on the VA Mobile team to standardize TTY language and affordances. While doing some research into another application, I took a look at our Facility Locator app. There's an opportunity to improve our search results by adding TTY links, using our [Telephone component](https://design.va.gov/storybook/?path=/docs/components-telephone--default) for deaf/hard-of-hearing users. Screenshots attached below.

## Acceptance Criteria

- [ ] TTY links are added to Facility Locator search results page using the `<va-telephone>` component
- [ ] TTY text is included in the linked text as indicated in [DS guidelines](https://design.va.gov/content-style-guide/dates-and-numbers#phone-numbers)
- [ ] Accessibility Lead review

## Environment
* [Austin, TX VA Health search](https://va.gov/find-locations/?page=1&address=Austin%2C%20TX&facilityType=health&serviceType&latitude=30.2711&longitude=-97.7437&radius=43&bounds%5B%5D=-98.4937&bounds%5B%5D=29.5211&bounds%5B%5D=-96.9937&bounds%5B%5D=31.0211&context=Austin%2C%20Texas%2C%20United%20States)

## Proposed Solution (if known)

```javascript
<div>
  // Previous content
  TTY: <Telephone contact="711" />
</div>
```

## WCAG or Vendor Guidance (optional)
* [Making actions keyboard accessible by using keyboard event handlers](https://www.w3.org/WAI/GL/wiki/Making_actions_keyboard_accessible_by_using_keyboard_event_handlers_with_WAI-ARIA_controls)

## Screenshots or Trace Logs
![Screen Shot 2021-03-10 at 3 58 09 PM](https://user-images.githubusercontent.com/934879/110708182-ec696980-81bf-11eb-96f6-5449e5f0cf21.png)

---
![Screen Shot 2021-03-10 at 4 31 45 PM](https://user-images.githubusercontent.com/934879/110708497-626dd080-81c0-11eb-9321-c9776b9c7c6d.png)

**Comment by laflannery** [comment]
Closing in favor of #15304
